### ISSUE TITLE
Importing a .pmx model with national characters in file path fails with an exception

### ISSUE THREAD

**Issue by shadowpanther** [Neos-Metaverse/NeosPublic#3036]
### Describe the bug?

When trying to import a .pmx model with file placed into a folder that contains national (Russian in this case) symbols in the path, Assimp fails to import the model with the following exception:
```
15:19:30.293 ( 60 FPS)	Importing files with Asset Class Model:
D:\rrrde\Загрузки\!MODELS\Tanuki_v1.0\raccoon_girl.pmx
15:19:38.711 ( 62 FPS)	Exception when importing D:\rrrde\Загрузки\!MODELS\Tanuki_v1.0\raccoon_girl.pmx:

Assimp.AssimpException: Error importing file: Failed to open file D:\rrrde\Загрузки\!MODELS\Tanuki_v1.0\raccoon_girl.pmx.
  at Assimp.AssimpContext.ImportFile (System.String file, Assimp.PostProcessSteps postProcessFlags) [0x00084] in <2bbd770ad2e943ec9a89007d8dfbcb77>:0 
  at FrooxEngine.ModelImporter+<ImportModel>d__6.MoveNext () [0x006c2] in <37c1b9afea074aa2a9d4fa3ffc975415>:0 
```

### Relevant issues

None found

### To Reproduce

Try to import a .pmx file placed in a folder with national characters in the name/path.
It is currently unknown what characters in particular cause this.

### Expected behavior

Model imports as usual

### Log Files

[log from demonzla](https://cdn.discordapp.com/attachments/826219085238763581/888839443149111336/DESKTOP-M0NR0IM_-_2021.9.16.105_-_2021-09-18_15_17_34.log)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.9.16.105

### What Platforms does this occur on?

Windows

### Did this work before?

I Don't Know

### Reporters

demonzla#4135

**Comment by jeanahelver** [comment]
is it only happening for pmx models? could you try fbx/obj.

**Comment by shadowpanther** [comment]
gltf, blend and fbx seem to load just fine, that's why I mentioned pmx in the report.

**Comment by Frooxius** [comment]
Do you have any sample file I could test with? I'm updating Assimp to latest version in the next build, but I can't verify if that helps so you'll have to check yourself. Otherwise we'll have to report this directly to Assimp to fix on their end.

**Comment by demonzla** [comment]
here two sample models in pmx
[1.zip](https://github.com/Neos-Metaverse/NeosPublic/files/7193561/1.zip)

**Comment by Frooxius** [comment]
Thanks! I've released latest Assimp in 2021.9.20.1334, but it doesn't seem like it fixes this issue. This means we'd have to make a report upstream.

Would anyone be willing to help with this? Alternatively anyone can implement a fix for this too if they can. Here's our fork for reference: https://github.com/Neos-Metaverse/assimp
